bin/scala-native-setup/macos-setup.sh

```diff
@@ -0,0 +1,3 @@
+apt-get install -y clang-12.0
+PATH=/usr/lib/llvm-12.0/bin:${PATH}
```

Comment:
Are you sure this is needed? I never had to install clang for native, I think it's included in the images.

Reply:
I guess that's a leftover from the previous iteration, I wonder if I even needed it then? Removing now

Reply:
maybe at some point?